patches/electron-winstaller@5.3.1.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/lib/index.js b/lib/index.js
+index d666a6316273b96cb67213e40e17c4a30625f63b..0e328caa0e33d13d6081f8aa42662bc28aaa7116 100644
+--- a/lib/index.js
++++ b/lib/index.js
+@@ -161,7 +161,7 @@ function createWindowsInstaller(options) {
+                     _f.label = 9;
+                 case 9:
+                     Object.assign(metadata, {
+-                        exe: "".concat(appMetadata.name, ".exe"),
++                        exe: "".concat(appMetadata.productName, ".exe"),
```

Comment:
What's this for? Can we continue using `name` in `options`? I see the `productName` is removed from options here: https://github.com/electron-userland/electron-builder/pull/8344/files#diff-0d44b799fbfefe8d2452d2b7395cb069788adcde5e199b3e83b6de88fc0c97a4L105

Reply:
`electron-winstaller` provides an exe parameter configuration, so I have removed this patch.